blueman/DeviceClass.py

```diff
@@ -180,8 +180,8 @@
     11: ('Barcode Scanner', {0: _("Generic Barcode Scanner")}),
     12: ('Thermometer', {0: _("Generic Thermometer"),
                          1: _("Thermometer: Ear")}),
-    13: ('Heart rate Sensor', {0: _("Generic Heart rate Sensor"),
-                               1: _("Heart Rate Sensor: Heart Rate Belt")}),
+    13: ('Heart rate Sensor', {0: _("Generic Heart-rate Sensor"),
```

Comment:
https://specificationrefs.bluetooth.com/assigned-values/Appearance%20Values.pdf says `Heart Rate`. I'd stick to that.

Reply:
True, but when there is a third word, is it "a heart-rate sensor", or a "heart rate-sensor". That is also why I added one for the lone "heart-rate", since that is consistent, and I don't think illegal.

Reply:
The spec says `Heart Rate Sensor`. Not saying that we absolutely need to follow it, of course, it just seems reasonable to me.